site/webpack.common.ts

```diff
@@ -103,6 +103,7 @@ export const createCommonWebpackConfig = (options?: { skipTypecheck: boolean }):
     //
     // See: https://webpack.js.org/guides/typescript/
     extensions: [".tsx", ".ts", ".js"],
+    modules: [path.resolve(__dirname, "src"), "node_modules"],
```

Comment:
We have three webpack files (common, prod, dev). It looks to me like we're splitting webpack config into different use cases, but I'm not sure. Can someone please confirm `webpack.common.ts` is an appropriate place for this change?